### PR TITLE
Improved 'Terms & Conditions' link in checkout

### PR DESCRIPTION
**Pull request by samwrigley**
### Changed

- Removed hard-coded path to 'Terms & Conditions' page in checkout and replaced with the `route` helper function.
- Ensured text inside 'Terms & Conditions' link in checkout is translatable.
- Added class to 'Terms & Conditions' link in checkout in order to style element more easily.
- Improved readability of 'Terms & Conditions' link markup in checkout.

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -5,7 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.4.0] - 2018-04-25
```
Comment (samwrigley) [inline changelog.md]: @m2de I've increased the package's version by a minor as other projects using this package will need add the `termsConditions` route.

File: src/resources/views/checkout/payment.blade.php [hunk 2]
```diff
@@ -44,7 +44,13 @@
                         </template>
                     </div>
                     <div class="checkout__terms">
-                        <label><input type="checkbox" v-model="currentCheckout.user.terms">@lang('Please check the box to agree to our') <a href="/pages/legal/terms-conditions-summary" target="_blank">terms &amp; conditions</a></label>
+                        <label>
+                            <input type="checkbox" v-model="currentCheckout.user.terms">
+                            @lang('Please check the box to agree to our')
+                            <a class="checkout__terms-link" href="{{ route('termsConditions') }}" target="_blank">
```
Comment (samwrigley) [inline src/resources/views/checkout/payment.blade.php]: @m2de, are you happy with using lower camel-case for the route naming convention?

Comment (m2de) [reply src/resources/views/checkout/payment.blade.php]: Whatever we currently do?

Comment (samwrigley) [reply src/resources/views/checkout/payment.blade.php]: Currently we use lower camel-case @m2de, but just thought I'd double check.

**Pull request merged**